likecoin/js/admin-settings/src/store/site-matters-store.js

```diff
@@ -36,12 +37,29 @@ const actions = {
       errorMsg,
     };
   },
+  * siteMattersLogin(options) {
+    try {
+      const response = yield { type: 'MATTERS_LOGIN', data: options };
+      return response;
+    } catch (error) {
+      console.error(error);
+      return error;
+    }
+  },
+  * siteMattersLogout() {
+    try {
+      const response = yield { type: 'MATTERS_LOGOUT' };
+      return response;
+    } catch (error) {
+      console.error(error);
+      return error;
+    }
+  },
   * postSiteMattersOptions(options) {
     yield { type: 'POST_SITE_MATTERS_OPTIONS_TO_DB', data: options };
     yield { type: 'CHANGE_SITE_MATTERS_OPTIONS_GLOBAL_STATE', data: options };
   },
-  * postSiteMattersLogin(user) {
-    yield { type: 'POST_SITE_MATTERS_USER_TO_DB', data: user };
+  * postSiteMattersLoginData(user) {
```

Comment:
Now this only update the local matters user name right? Please rename to `update` matters user name or sth without post

Reply:
Yes, will change in next fix.